context-free.md

```diff
@@ -1,22 +1,22 @@
 # Context Free
 
 ##  Who is this programming language for?
-
++ This language is for people who want to programatically create images. 
 
 ## What is easy to do in this language? Why is it easy?
-
++ It is easy to draw images that are made up of many much simpler shapes such as squares, circles etc. This is easy because the definitions for these shapes are already available in the language. 
```

Comment:
I would additionally mention that its easy to make these shapes defined by recurrence relations, so its not just that its easy to make lots of the same triangle.